src/modules/pushPair/services/PushPairRequestService.test.ts

```diff
@@ -0,0 +1,113 @@
+import Big from "big.js";
+import { createDataRequestMock } from "../../../mocks/DataRequestMock";
+import { PushPairDataRequest } from "../models/PushPairDataRequest";
+import { shouldPricePairUpdate } from "./PushPairRequestService";
+
+describe('PushPairRequestService', () => {
+    describe('shouldPricePairUpdate', () => {
+        it('should return true if there is no oldPrice', () => {
+            const dr = createDataRequestMock({
+                extraInfo: {
+                    deviationPercentage: 10,
+                    minimumUpdateInterval: 3600000, // 1 hour
```

Comment:
Define const in a higher scope? :P 
```suggestion
                    minimumUpdateInterval: oneHour,
```